.github/workflows/rust.yml

```diff
@@ -31,16 +31,25 @@ concurrency:
 jobs:
   test:
     if: github.event.pull_request.draft == false
-    runs-on: buildjet-4vcpu-ubuntu-2204
+    runs-on: ubuntu-latest
```

Comment:
fyi the github runner ran out of disk space in `rust.yml`, `js.yml`, `light-sdk-tests.yml`, `light-system-programs-test.yml`.